src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -25,7 +25,7 @@
     DECREASE_TABLE_FONT_SIZE("Decrease table font size", Localization.lang("Decrease table font size"), "ctrl MINUS"),
     DELETE_ENTRY("Delete entry", Localization.lang("Delete entry"), "DELETE"),
     EDIT_ENTRY("Edit entry", Localization.lang("Edit entry"), "ctrl E"),
-    EDIT_PREAMBLE("Edit preamble", Localization.lang("Edit preamble"), "ctrl P"),
+    EDIT_PREAMBLE("Edit preamble", Localization.lang("Edit preamble"), "alt S"),
```

Comment:
This change is not shown at your overview https://github.com/JabRef/jabref/pull/1525#issue-162928053

Furthermore, `p` sounds more like **p**reamble than `s`.. Why was it changed?

Furthermore, `alt`+ letter is more accessing menus. What if the menu shortcut in some other language is `s`?